### PR TITLE
Add Web Notifications Types

### PR DESCRIPTION
**Pull request by rposener**
Please fill in this template.

- [Y ] Prefer to make your PR against the `types-2.0` branch.
- [Y] Test the change in your own code.
- [Y ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [Y ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [ ] Run `npm run lint -- package-name` if a `tslint.json` is present.

If adding a new definition:
- [Y ] The package does not provide its own types, and you can not add them.
- [Y ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [Y ] Run `tsc` without errors.
- [Y ] Include the required [files](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package) and header. Base these on the README, *not* on an existing project.

If changing an existing definition:
- [ ] Provide a URL to  documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.

### CODE REVIEW

**Comment by andy-ms** [comment]
This doesn't follow the `types-2.0` format. Could you use `npm run new-package -- webnotifications` to generate the files, then fill them in?

File: webnotifications/webnotifications.d.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+    /**
+     * The title shown in the notification
+     * @readonly
+     */
+    title?: string;
+
+    /**
+     * The notification language
+     * @readonly
```
Comment (andy-ms) [inline webnotifications/webnotifications.d.ts]: You can use the `readonly` keyword to specify a readonly property. `readonly lang?: string;`.

**Comment by rposener** [comment]
Before I start revising, should this be in types-2.0?  Or should I move this to the master branch?  This is a ES5 spec that is missing / not available.  Really I would have expected it to be in types.d.ts with the typescript install for ES2015 target.

**Comment by rposener** [comment]
Just looking to have this spec available: https://www.w3.org/TR/notifications/

**Comment by andy-ms** [comment]
I think this is more appropriate for [TSJS-lib-generator](https://github.com/Microsoft/TSJS-lib-generator) then. Please file an issue (or a PR) there and reference this.

**Pull request closed without merging**